src/containers/TemplateBuilder/shared/OperationContextHelpers.tsx

```diff
@@ -225,6 +225,11 @@ export const duplicateTemplate: TemplateOperationHelper = async (
     body,
     setErrorAndLoadingState
   )
+
+  // Delete the snapshot cos we don't want snapshots page cluttered with individual templates
+  safeFetch(getServerUrl('snapshot', { action: 'delete', name: snapshotName }), {}, () => {})
+
+  return snapshotResult
```

Comment:
This is exactly the same as the change I made to the "importTemplate" method a while back (just below this one)

I should have done it at the same time, but alas I forgot about duplication as well as import.